README.md

```diff
@@ -191,9 +189,29 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 | hang          | Object | No       | { "enabled": true } | Generate hang diagnostic as log records.                                                                                                                                                                  |
 | view          | Object | No       | { "enabled": true } | Create spans from views created with UIKit and SwiftUI.                                                                                                                                                   |
 
-## Auth
+## Export Strategy
+
+The AWS OpenTelemetry Swift SDK uses OTLP (OpenTelemetry Protocol) over HTTP to export telemetry data to AWS CloudWatch RUM. The SDK supports multiple authentication methods and export configurations.
+
+### Default Export Behavior
+
+By default, the SDK exports telemetry data with unsigned requests, which is the recommended pattern for monitoring real users. Please refer to AWS docs for allowing [unauthenticated ingestion via RUM resource based policy](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-RUM-resource-policies.html).
 
-Generally, [unauthenticated ingestion via RUM resource based policy](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-RUM-resource-policies.html) is the recommended pattern for monitoring real users. However, we also support using custom identity provider scenarios with the `AwsOpenTelemetryAuth` module:
+| Field                | Type              | Required | Default                   | Description                                                       |
+| -------------------- | ----------------- | -------- | ------------------------- | ----------------------------------------------------------------- |
+| maxRetries           | `Int`             | No       | 3                         | Maximum number of retry attempts                                  |
+| retryableStatusCodes | `Set<Int>`        | No       | [429, 500, 502, 503, 504] | HTTP status codes that trigger retry logic                        |
```

Comment:
question (non blocking): should we be expanding the retryable status codes to those listed here? 
https://docs.aws.amazon.com/sdkref/latest/guide/feature-retry-behavior.html